core/cas-server-core-configuration-api/src/main/java/org/apereo/cas/configuration/DefaultCasConfigurationPropertiesSourceLocator.java

```diff
@@ -58,11 +61,9 @@
             LOGGER.info("Configuration directory [{}] is not a directory or cannot be found at the specific path", config);
         }
 
-        final File configFile = casConfigurationPropertiesEnvironmentManager.getStandaloneProfileConfigurationFile();
-        if (configFile != null) {
-            final PropertySource<?> sourceStandalone = loadSettingsFromStandaloneConfigFile(configFile);
-            compositePropertySource.addFirstPropertySource(sourceStandalone);
```

Comment:
Yes it was, but the configurationDirectory was lower priority than the classpath, so I rearranged the property sources so that they were simply added to the compositePropertySource in priority order.